### PR TITLE
WIP: Rispy inclusion

### PR DESCRIPTION
**Pull request by ottomattas**
This pull request is Work In Progress

### CODE REVIEW

**Comment by ottomattas** [comment]
There are some temporarily commented lines (e.g asreview/io/ris_reader.py) which should be removed before merging this branch into the main in upstream.

File: tests/demo_data/generic_labels.ris [hunk 1]
```diff
@@ -22,6 +22,7 @@ SN  - 1932-6208
 M1  - 1008150341
 L2  - http://example.com
 UR  - http://example_url.com
+N1  - ASReview_relevant
 LI  - 1
```
Comment (J535D165) [inline tests/demo_data/generic_labels.ris:26]: Remove this one?

Comment (ottomattas) [reply tests/demo_data/generic_labels.ris]: I reverted the initial commit to adjust the file: https://github.com/asreview/asreview/pull/603/commits/d026094cb1b7e3a4070b1354c4744166260638c3

**Comment by J535D165** [comment]
Closing this PR in favor of the new PR/implementation (in agreement with @ottomattas). Nevertheless, thanks for the effort.

**Pull request closed without merging**